File: packages/sdks/javascript/src/services/OrganizationService.ts
import { OrganizationsApiService } from "../api/generated/apis/organizations-api";
import { OrganizationsEnvironmentsApiService } from "../api/generated/apis/organizations-environments-api";

export class OrganizationService extends OrganizationsApiService {
  environments = new OrganizationsEnvironmentsApiService();
}

File: packages/sdks/python/src/devopness/services/organization_service.py
"""
Devopness API Python SDK - Painless essential DevOps to everyone
"""

from ..generated.api.organizations_api import (
    OrganizationsApiService,
    OrganizationsApiServiceAsync,
)

__all__ = ["OrganizationService", "OrganizationServiceAsync"]


class OrganizationService(
    OrganizationsApiService,
):
    """Service for organizations in the Devopness API."""


class OrganizationServiceAsync(
    OrganizationsApiServiceAsync,
):
    """Async service for organizations in the Devopness API."""
